style.css

```diff
@@ -997,8 +1003,1095 @@ h2 {
+  gap: 10px;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  padding: 0 20px;
+  transform: translate(-50%, -50%) scale(0);
+  transition: 200ms ease-in-out;
+  z-index: 10;
+  background: #fff;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  border-radius: 16px;
+  width: 400px;
+  max-width: 70%;
+  height: 95%;
+}
+
+.model.active {
+  transform: translate(-50%, -50%) scale(1);
+}
+
+#overlay {
+  position: fixed;
+  opacity: 0;
+  transition: 200ms ease-in-out;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background: #413c3b;
+  mix-blend-mode: multiply;
+  pointer-events: none;
+}
+
+#overlay.active {
+  opacity: 1;
+  pointer-events: all;
+}
+
+.model-img {
+  width: 80%;
+  height: 200px;
+}
+
+.popupContainer {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 0;
+  gap: 5px;
+  width: 311px;
+  height: 10px;
+}
+
+.h2popup {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  text-align: left;
+}
+
+.close {
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: none;
+  color: #67798e;
+  font-size: 30px;
+}
+
+.popuptonic {
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  padding: 0;
+  align-items: center;
+}
+
+.popuptonic.canopy {
+  width: 55px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.popuptonic.back {
+  width: 90px;
+  height: 50px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.popuptonic.dot {
+  margin-top: 0;
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  border-radius: 50%;
+}
+
+.popuptonic.year {
+  width: 29px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.model-body {
+  gap: 0;
+}
+
+.popupparagraph2 {
+  width: 330px;
+  height: 160px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 15px;
+  color: #344563;
+}
+
+.popupresources {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: flex-start;
+  padding: 0;
+  gap: 8px;
+  width: 203px;
+  height: 24px;
+}
+
+.popupbutton {
+  height: 20px;
+  background-color: hsl(240, 72%, 94%);
+  border: none;
+  color: #212658;
+  text-align: center;
+  text-decoration: none;
+  padding: 4px 12px;
+  align-items: center;
+  cursor: pointer;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 8px;
+  line-height: 16px;
+  letter-spacing: 0.01em;
+}
+
+.popupaction {
+  margin-top: 5px;
+  display: flex;
+  gap: 12px;
+}
+
 @media screen and (min-device-width: 768px) {
-  header {
+  .logo {
```

Comment:
- Great Job implementing your desktop version. Everything looks great on screen size 1440px, Also, everything on mobile is perfect. 👍🏽 
- I'd like to kindly suggest that you fix your portfolio on tablet screens (768px). The Project cards, about section is squashed up, and the contact section too.
- [ ] Suggestion: I think you can have 2 media queries. You can replace 768px on line 1180 with 1024px, and then write another media query specifically for 768px. This should work.

Here is what I could see on the browser 👇🏽 

| Your Project card  |
| :---:   | 
|<img width="590" alt="Screenshot 2023-02-22 at 17 39 42" src="https://user-images.githubusercontent.com/99246359/220789535-661f150f-11e1-4cfc-bfb6-d7681cc87166.png">  |

|About Page|
| :---:   | 
|<img width="578" alt="Screenshot 2023-02-22 at 17 40 07" src="https://user-images.githubusercontent.com/99246359/220789563-0a081099-9df5-41ec-b80a-1206330e0625.png">  |


|contact form |
| :---:   | 
|<img width="590" alt="Screenshot 2023-02-22 at 17 40 29" src="https://user-images.githubusercontent.com/99246359/220789577-b25a508b-f562-4844-b5d5-2eccfc072442.png">  |